### PR TITLE
add Rests Documentation

### PR DESCRIPTION
**Pull request by sienna-kopf**
# SOM-Timer BE Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- update documentation page at the root to include rest index and creation endpoints

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs/pry sessions
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Som Timer Backend 
+# Som Timer Backend
 
 See [Som Timer Frontend](https://github.com/SOM-Timer/som_timer_fe) for more information!
```
Comment (jkwest-93) [inline README.md:3]: Thanks for the shoutout!!!

**Review by rwilliams659 — APPROVED** [review]
Great job!! This is looking really clean now

**Review by jkwest-93 — APPROVED** [review]
Thanks for adding this documentation on the session data!

**Review by sciencefixion — APPROVED** [review]

**Pull request merged**